src/bin/index.js

```diff
@@ -5,33 +5,49 @@ const yargs = require("yargs")
 const { downloadCourses } = require("../aws_sync")
 const { scanCourses } = require("../file_operations")
 
+const { MISSING_JSON_ERROR_MESSAGE } = require("../constants")
+const loggers = require("../loggers")
+
 // Gather arguments
 const options = yargs
   .usage("Usage: -s <path> -d <path>")
   .option("c", {
     alias:        "courses",
-    describe:     "A JSON file describing courses to download from AWS",
+    describe:     "A JSON file describing courses to process",
     type:         "string",
     demandOption: false
   })
-  .option("s", {
-    alias:        "source",
-    describe:     "Source directory of courses",
+  .option("d", {
+    alias:    "download",
+    describe:
+      "A flag that if set to true to will download courses passed in from AWS",
+    type:         "boolean",
+    demandOption: false
+  })
+  .option("i", {
+    alias:        "input",
+    describe:     "Input directory of courses",
     type:         "string",
     demandOption: true
   })
-  .option("d", {
-    alias:        "destination",
-    describe:     "Destination directory for Hugo markdown",
+  .option("o", {
+    alias:        "output",
+    describe:     "Output directory for Hugo markdown",
     type:         "string",
     demandOption: true
   }).argv
```

Comment:
it's too bad `yargs` doesn't support things like a `dir` and `file` type, which could automatically check if they're present and readable / writable

Reply:
Yea, unfortunately it only supports array, boolean or string